core/tests/protractor_utils/forms.js

```diff
@@ -249,9 +250,15 @@ var RichTextEditor = async function(elem) {
       }
       await richTextComponents.getComponent(componentName)
         .customizeComponent.apply(null, args);
-      await modal.element(
-        by.css('.protractor-test-close-rich-text-component-editor')).click();
-
+      var doneButton = await modal.element(
```

Comment:
You shouldn't need an `await` here since `element(...)` returns an `ElementFinder`, not a `Promise`

Reply:
done